oc-cluster

```diff
@@ -253,7 +303,15 @@ function up {
 
     echo "$CMDLINE" > $OPENSHIFT_PROFILES_DIR/$_profile/run
     echo "$(<$OPENSHIFT_PROFILES_DIR/$_profile/run)"
-    $CMDLINE
+    if [ ! -z $DEV_DOCKER_HOST ]; then
+      echo "Running in developer mode, using DOCKER_HOST=$DEV_DOCKER_HOST"
+      if [[ "$__PLATFORM" == "linux" ]]; then
+        CMDLINE="DOCKER_HOST=$DEV_DOCKER_HOST $CMDLINE"
+      fi
+      $CMDLINE -e DOCKER_HOST=$DEV_DOCKER_HOST
```

Comment:
I'll look into this as again this is Linux specific. Can you PR a fix as you'll probably get it better than me?

Have you tested it?

Also I would love the mounts-template.json completed for injecting openshift binary.